### PR TITLE
add caseFormat as a new study info

### PR DESCRIPTION
**Pull request by ChamseddineBhd**
Signed-off-by: ChamseddineBhd <chamseddine.benhamed@ensi-uma.tn>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/test/java/com/powsybl/study/server/StudyTest.java [hunk 1]
```diff
@@ -97,6 +97,18 @@ public void setup() {
         studyService.setSingleLineDiagramServerRest(singleLineDiagramServerRest);
         studyService.setGeoDataServerRest(geoDataServerRest);
 
+        given(caseServerRest.exchange(
```
Comment (geofjamg) [inline src/test/java/com/powsybl/study/server/StudyTest.java]: This one is not used?

Comment (ChamseddineBhd) [reply src/test/java/com/powsybl/study/server/StudyTest.java]: it's used

Comment (ChamseddineBhd) [reply src/test/java/com/powsybl/study/server/StudyTest.java]: studyTest when inserting a study we should call that method : line 188

Comment (geofjamg) [reply src/test/java/com/powsybl/study/server/StudyTest.java]: ok

**Pull request merged**